server/cluster/cluster.go

```diff
@@ -108,3 +109,7 @@ func (c *Cluster) GetShards() []storage.ShardID {
 func (c *Cluster) GetShardNodes() metadata.GetShardNodesResult {
 	return c.metadata.GetShardNodes()
 }
+
+func getProcedureIDPath(rootPath string) string {
```

Comment:
`make` or `format` seems better than `get`:
```suggestion
func makeProcedureIDPath(rootPath string) string {
```